e2e/test-plugins/grafana-extensionstest-app/pages/AddedLinks.tsx

```diff
@@ -1,25 +1,22 @@
 import { PluginPage, usePluginLinks } from '@grafana/runtime';
+import { Stack } from '@grafana/ui';
 
+import { ActionButton } from '../components/ActionButton';
 import { testIds } from '../testIds';
 
 export const LINKS_EXTENSION_POINT_ID = 'plugins/grafana-extensionstest-app/use-plugin-links/v1';
 
 export function AddedLinks() {
-  const { links, isLoading } = usePluginLinks({ extensionPointId: LINKS_EXTENSION_POINT_ID });
+  const { links } = usePluginLinks({ extensionPointId: LINKS_EXTENSION_POINT_ID });
```

Comment:
`isLoading` wasn't used in the test. Writing an e2e test for this could be tricky, as extensions are registered so quickly (even though they're now reactive). Maybe we should revisit this when we're adding support for lazy loading apps when its extensions are being used? Then the `isLoading` indicator could actually have a more central role. WDYT?

Reply:
Yepp, good point, let's do this in the lazy-loading PR 👍